prime-dotnet-webapi/Controllers/SubmissionsController.cs

```diff
@@ -215,6 +215,23 @@ public async Task<ActionResult> RerunRules(int enrolleeId)
             return await EnrolleeStatusActionInternal(enrolleeId, EnrolleeStatusAction.RerunRules);
         }
 
+        // POST: api/enrollees/rerun-rules
+        /// <summary>
+        /// Re-runs the automatic adjudication rules for all applicable Enrollees.
+        /// </summary>
+        [HttpPost("rerun-rules", Name = nameof(RerunAutomaticAdjudicationRules))]
+        [Authorize(Roles = Roles.PrimeApiServiceAccount)]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status200OK)]
```

Comment:
Many of these Swagger annotations are incorrect

Reply:
Should be fixed now